Sources/ADNetPromoterScoreSurvey/ADNetPromoterScoreSurvey.swift

```diff
@@ -71,17 +71,12 @@ import UIKit
     }
     
     fileprivate func initSurveyView(){
-        
-        let bundle = Bundle(for: ADNetPromoterScoreSurvey.self)
-        let nib = UINib(nibName: "NPSSurveyView", bundle: bundle)
-        
-        if let surveyView = nib.instantiate(withOwner: nil, options: nil).first as? NPSSurveyView{
-            
-            surveyView.translatesAutoresizingMaskIntoConstraints = false
+
+        let surveyView = NPSSurveyView()
+        surveyView.translatesAutoresizingMaskIntoConstraints = false
             
-            self.surveyView = surveyView
-            self.surveyView!.delegate = self
-        }
+        self.surveyView = surveyView
+        self.surveyView!.delegate = self
```

Comment:
`!` or `?`

Reply:
good point. this line merely changes indentation - it was force unwrapping before. But yeah we should remove force unwraps everywhere if we are to use this component